src/job/jobHandler.ts

```diff
@@ -234,21 +234,94 @@ export abstract class JobHandler {
     }
   }
 
+  private async loggingMessage(resp: CommandExecutorResponse): Promise<void> {
+    await this._logger.save(this.currJob._id, `${'(BUILD)'.padEnd(15)}Finished Build`);
```

Comment:
do we need to input `finished build` for each time we log? seeing this multiple times on the sample log in description. i think we can log these in the exeBuildModified after all jobs have executed

Reply:
@seungpark we could, good call out. I can move this out so it isn't logged each time a build step is done.